cypress/e2e/article.cy.js

```diff
@@ -1,13 +1,43 @@
-describe('', () => {
-  before(() => {
 
-  });
+describe('Article', () => {
+  let user;
+  let articleTest;
+
+  beforeEach(() => {
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+    });
 
-  it('', () => {
+    cy.task('generateArticle').then((generateArticle) => {
+      articleTest = generateArticle;
+    });
 
+    cy.visit('');
   });
 
-  it('', () => {
+  it('The article should be created', () => {
+    cy.login(user.email, user.username, user.password);
```

Comment:
There is no test to check if the login function is working correctly before creating or deleting an article. Please add a test for it.